node/engine/chainservice/mock_chain.go

```diff
@@ -33,21 +33,24 @@ func NewMockChain() *MockChain {
 // unlike an ethereum blockchain, MockChain accepts go-nitro protocols.ChainTransaction
 func (mc *MockChain) SubmitTransaction(tx protocols.ChainTransaction) error {
 	mc.blockNum++
+	channelIdString := tx.ChannelId().String()
+	h, _ := mc.holdings.Load(channelIdString)
```

Comment:
Swallowed `ok`s or `error`s raise my eyebrows and slow my reading of code. A quick disclaimer may be a good idea when the swallowing isn't immediately & obviously correct.

Maybe

```suggestion
	h, _ := mc.holdings.Load(channelIdString) // ignore `ok` because the returned zero-value is what we want
```

In this case it is correct because:
- safesync.Map returns a zero-valued `T` (types.Funds in this case) when its lookup fails
- the zero-value of `types.Funds` is well behaved with its `Add` function

Reply:
Great point.